README.md

```diff
@@ -351,6 +351,8 @@ The CLI does not expose everything in the SDK, but it is great for doing simple
 $ oneview-sdk-ruby list ServerProfiles
```

Comment:
adding a comment here:  outputs a list of resource names

Reply:
Done